app/views/conferences/_conference_details.html.haml

```diff
@@ -34,6 +34,8 @@
               = link_to "My Proposals", conference_program_proposals_path(conference.short_title), class: 'btn btn-default'
             - elsif can? :new, conference.program.events.new
               = link_to "Submit Proposal", new_conference_program_proposal_path(conference.short_title), class: 'btn btn-default'
+            - if current_user && current_user.ticket_purchases.by_conference(conference).unpaid.any?
```

Comment:
in the conference details we could a link to the pyshical tickets index page with the text `Tickets`.

And as we want to have message with a link from the pysical tickets index page to the unpaid ones, in case that you want to reach the unpaid ones, you would have to go to tickets and then to the unpaid tickets. To avoid having many things in the conference details.

Did I explain myself better now? :wink: